.github/actions/k8s-compatibility-test/action.yaml

```diff
@@ -24,10 +24,10 @@ runs:
         go-version: "1.21"
     - name: Checkout to PR branch # to remove after getting rid of pull_request_target
       shell: bash
-      if: ${{ github.event.type == 'PullRequestEvent' }}
+      if: github.event_name == 'pull_request' || github.event_name == 'pull_request_target'
```

Comment:
I think that there might be the issue that until this is merged it will still use the old implentation from base (release-2.0) branch

Reply:
yes

Reply:
will redo the branch :(